packages/data/src/index.ts

```diff
@@ -29,12 +39,12 @@ export { default as createReduxStore } from './redux-store';
 export { dispatch } from './dispatch';
 export { select } from './select';
 
+export * from './types';
```

Comment:
This is the only part that needs a second opinion. I am not sure if we want to expose all the types here.

Reply:
I don't know but I'm also not very concerned either, because we could change our decision later. It's not a public API in the sense of a WordPress public API. It's only a public API to npm consumers (where breaking changes are allowed)